package.json

```diff
@@ -140,5 +140,8 @@
     "webpack": "5.50.0",
     "webpack-bundle-analyzer": "4.4.2",
     "webpack-cli": "4.7.2"
+  },
```

Comment:
@charellesandig Adding the dependency here will not update it in the specific child packages where it is being declared. Instead of adding it here as an additional declaration, what you want is to update the existing declarations that are pointing to out of date versions. You can use `npm ls morgan` to find out where are those.

One more thing: Please always use exact versions and lock them down without any sort of auto-upgrade.

```sh
$ npm ls morgan
@hyperledger/cactus@ /home/peter/a/blockchain/cactus
├─┬ @hyperledger/cactus-cmd-socket-server@1.0.0-rc.3 -> ./packages/cactus-cmd-socketio-server
│ └── morgan@1.9.1
├─┬ @hyperledger/cactus-plugin-ledger-connector-fabric-socketio@1.0.0-rc.3 -> ./packages/cactus-plugin-ledger-connector-fabric-socketio
│ └── morgan@1.8.2
├─┬ @hyperledger/cactus-plugin-ledger-connector-go-ethereum-socketio@1.0.0-rc.3 -> ./packages/cactus-plugin-ledger-connector-go-ethereum-socketio
│ └── morgan@1.8.2 deduped
└─┬ @hyperledger/cactus-plugin-ledger-connector-sawtooth-socketio@1.0.0-rc.3 -> ./packages/cactus-plugin-ledger-connector-sawtooth-socketio
  └── morgan@1.8.2 deduped
```